Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ManifoldsBase"
 uuid = "3362f125-f0bb-47a3-aa74-596ffd7ef2fb"
 authors = ["Seth Axen <seth.axen@gmail.com>", "Mateusz Baran <mateuszbaran89@gmail.com>", "Ronny Bergmann <manopt@ronnybergmann.net>", "Antoine Levitt <antoine.levitt@gmail.com>"]
-version = "0.9.2"
+version = "0.9.3"
```

Comment:
I think this is breaking enough to call it 0.10.0.

Other than that, I think this is good to go as soon as you are happy with how it works with https://github.com/JuliaManifolds/Manifolds.jl/pull/216

Reply:
It basically only breaks said PR - but to be honest, that's the first time we actually use the embedded manifold. I checked with 216 and I am fine with that. So we can surely merge this.

I don'T have a strong preference whether to call it `0.9.3` or `0.10.0` so I leave the decision up to you.

Reply:
I think we can also call it a technically breaking bugfix. I'll leave 0.9.3 then.